src/components/dashboard/Dashboard.js

```diff
@@ -175,8 +176,11 @@ const Dashboard = props => {
     if (res.length == 0) {
       return
     }
+
     if (reset) {
+      // a flag used to show feed load animation only at the first app loading
       if (!loadAnimShown) {
+        // a time to perform feed load animation till the end
```

Comment:
are there some way to await when animation exactly ends ? Something like addEventListener("animationend" in DOM ?

Using delays is a bad idea as real animation time depends from many conditions (OS, browser app and its version, your CPU, graphics card, RAM amount, system load etc)